Documentation/Usage.md

```diff
@@ -115,204 +115,257 @@ url.
 Example of a published package:
 https://github.com/apple/example-package-fisheryates
 
-## Require System Libraries
+## Import system libraries
 
-You can link against system libraries using the package manager. To do so, there
-needs to be a special package for each system library that contains a modulemap
-for that library. Such a wrapper package does not contain any code of its own.
+As of Swift 4.2, the package manager is capable of importing C libraries from the system without requiring a special wrapper package. We’ll import [Cairo](cairographics.org), a popular 2D vector graphics library, as an example. This guide assumes you already know how to include and link a system library in a C or C++ project.
```

Comment:
Can we use the libgit example instead? I think more people will be familiar with the libgit library.

Reply:
is there really a difference? i’ve used cairo dozens of times but i can’t remember the last time i used libgit. maybe you link against it a lot because it gets used a lot when making development tools? either way i don’t think “library X is a better example than Y” when they both demonstrate the same thing is a good enough reason to rewrite the whole thing…